linera-service/src/linera.rs

```diff
@@ -610,6 +610,10 @@ struct ClientOptions {
     #[structopt(long, default_value = "1000")]
     cache_size: usize,
 
+    /// Create a table if one is missing
+    #[structopt(long = "create a database if missing", default_value = "true")]
+    create_if_missing: String,
```

Comment:
It could.
But I want the default to be `true`. And looking at the documentation, I could not find a way to do this.

Reply:
Ah, I see. :confused:
We could call it `dont_create_missing_table: bool`, so it would be `false` by default.
But I'm happy with the string, too, if you prefer.